build.gradle

```diff
@@ -6,6 +6,10 @@ plugins {
 group 'no.unit.nva'
 version '1.0-SNAPSHOT'
 
+repositories {
+    mavenLocal()
```

Comment:
why are we using mavenLocal here?

Reply:
I added it be able to develop against nva-commons before it was published on central. I will remove it.